src/modules/printUtils.js

```diff
@@ -9,6 +9,22 @@ export const a4LandscapeWidthPx = 1102
 
 export const MAX_ITEM_GRID_HEIGHT = 33
 
+export const getTransformYPx = elStyle => {
+    if (!elStyle || !elStyle.transform) {
+        return null
+    }
+
+    const transformY = elStyle.transform.split(' ')[1]?.slice(0, -1)
```

Comment:
Maybe a regexp can be used here to simplify a bit as it takes care of verifying that the format is correct:
```suggestion
   const transformY = elStyle.transform.split(' ')[1].match(/(\d+)px$/)
    if (transformY) {
      return parseInt(transformY[1])
    } else {
      return null
    } 
```